mne/decoding/classifier.py

```diff
@@ -453,3 +453,23 @@ def transform(self, epochs_data, y=None):
                                      picks=self.picks, n_jobs=self.n_jobs,
                                      copy=False, verbose=False)
         return epochs_data
+
+
+def compute_patterns(epochs, linear_model):
```

Comment:
This can be a nice unit test. First, fit a CSP and compute weights and patterns in the traditional way (inverse(W).T). Then compare with the patterns produced by this method. If the code is correct, they should be the same.